src/GT.Data/Repositories/Impl/GTIdentityRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using GT.Data.Data.GTIdentityDb;
```

Comment:
Should we really keep this file? Was under impression we skip this until we maybe decide to implement Middleware for identity and policies.

Reply:
It was decided that we keep the repository for future implementation of a user service.